docs/app/import-export.md

```diff
@@ -0,0 +1,125 @@
+[Items](/getting-started/glossary/#items), and [Fields](/getting-started/glossary/#fields).
+
+:::
+
+:::tip Import / Export via the API
+
+This page details Importing and Exporting of Items as files via the no-code app. However, you can also
+[Import](/reference/system/utilities/#import-data-from-file) and
+[Export](/reference/system/utilities/#export-data-to-a-file) Items as files programmatically via the API.
+
+:::
+
+## Import Items
+
+<video autoplay muted loop controls title="Import Data From a File">
+	<source src="" type="video/mp4" />
+</video>
+
+To import Items from a file, follow the steps below.
+
+1. Navigate to the desired Module and Collection.\
+   Note: the User Directory and File Library are each composed of one Collection.
+2. Click **"Import / Export"** in the Sidebar.
+3. Click into the import search box. A file browser will open.
+4. Select the desired file and click **"open"**.
+5. Click **"Start Import"** to import the Items.
+
+The new Items will now be in the Collection. The file itself will not be saved in the Directus Project.
+
+:::tip Importing CSV Files
+
+Any CSV column header that does not match a Field Name in the Collection will be ignored. If two columns in the CSV have
+the same header, then _(moving from left-to-right)_ the values from the second column will be used to create the new
+Items.
+
+:::
+
+:::tip Importing JSON Files
+
+Any value from a key that does not match a Field Name in the Collection will be ignored. If a JSON object has two keys
+with the same value, the second key-value pair _(moving from top-to-bottom)_ will be used to create the new Items.
+
+:::
+
+## Export Items
+
+<video autoplay muted loop controls title="Export Data to a File">
+	<source src="" type="video/mp4" />
+</video>
+
+When exporting Items, the [Export Items Menu](#export-items-menu) provides granular control over exactly which Items and
+Fields are exported, how they are exported, and where they are exported. To export Items, follow the steps below.
+
+1. Navigate to the desired Module and Collection.\
+   Note: the User Directory and File Library are each composed of one Collection.
+2. Optional: Configure the dataset as desired _(e.g. layout, options, search, filters, order, etc.)_.
+3. Click **"Import / Export"** in the Sidebar
+4. Click **"Export Items"** and the [Export Items Menu](#export-items-menu) will appear.
+5. Choose the desired format: CSV, JSON or XML.
+6. Configure any other export details as desired.
+7. Click <span mi btn>download</span> to download the file.
+
+:::tip Exporting to File Library
+
+When downloading 2,500 or more Items, you will be required to download into the Directus Project's File Library. You can
+also choose to download to the File Library in the [Export Items Menu](#export-items-menu). In either case, you can
+navigate to the File Library to download your file locally.
+
+:::
+
```

Comment:
Am I missing anything special/peculiar about csv/json/xml exports?

Reply:
Don't think so! The one thing we might want to highlight in a `::: tip` is that you should choose `csv` if you want to open it in Excel, as I would assume that's a common use case that not every user might be aware of

Reply:
Yeah, true!